Common/src/Pollster/Pollster.cs

```diff
@@ -120,8 +124,6 @@ await Init(cancellationToken)
         await foreach (var message in messages.WithCancellation(cancellationToken)
```

Comment:
Why `message` is not "used" anymore (ie: `using`). When is it disposed?

Reply:
It is explicitly disposed in the TaskHandler Dispose function